jep/1/README.adoc

```diff
@@ -0,0 +1,593 @@
+  syntax and semantics of any new feature. The specification should be
+  sufficiently detailed to allow new or existing Jenkins developers to
+  reasonably understand the scope/impact of an implementation.
+. **Motivation** - The motivation is critical for JEPs that want to
+  change Jenkins itself. It should clearly explain why the
+  existing code base is inadequate to address the
+  problem that the JEP solves. JEP submissions without sufficient
+  motivation may be rejected outright.
+. **Rationale** - The rationale fleshes out the specification by
+  describing what motivated the design and why particular design
+  decisions were made. It should describe alternate designs that
+  were considered and related work, e.g. how the feature is supported
+  in other languages.
++
+The rationale should provide evidence of consensus within the
+community and discuss important objections or concerns raised
+during discussion.
+
+. **Backwards Compatibility** - All JEPs that introduce backwards
+  incompatibilities must include a section describing these
+  incompatibilities and their severity. The JEP must explain how the
+  author proposes to deal with these incompatibilities. JEP submissions that do
+  not adequately discuss backwards compatibility, when such discussion is
+  required, may be rejected outright.
+. **Reference Implementation** -- The reference implementation must be
+  completed before any JEP is given status "Final", but it need not
+  be completed before the JEP is accepted. While there is merit
+  to the approach of reaching consensus on the specification and
+  rationale before writing code, the principle of "rough consensus
+  and running code" is still useful when it comes to resolving many
+  discussions of API details.
+. **References** -- When moving a JEP from a Draft to Accepted or Final state,
+  the references section should be updated to include links to the pull requests
+  and mailing list discussions which were involved in the process. The JEP
+  should self-document the process in which it was developed.
+
+The final implementation must include test code and documentation
+appropriate for either the Jenkins user or developer documentation.
+
+
+==== JEP Formats and Templates
+
+JEPs are UTF-8 encoded text files using the
+link:https://asciidoctor.org[AsciiDoc] format.  AsciiDoc allows for rich markup
+that is still quite easy to read, but also results in good-looking and
+functional HTML.
+
+
+==== JEP Header Preamble
+
+Each JEP must begin with an AsciiDoc table containing metadata relevant to the
+JEP:
+
+[source,asciidoc]
+----
+.Metadata
+[cols="2"]
+|===
+| JEP
+| 1
+
+| Title
+| Jenkins Enhancement Proposal Format
+
+| Author
+| link:https://github.com/rtyler[R Tyler Croy]
+
+| Status
+| :speech_balloon: Draft
+
+| Type
+| Process
+
+| Created
+| 2017-09-12
+|===
+----
+
+
+. **JEP** -- Proposal number, given by the JEP editors. Use "9999" until one is assigned.
```

Comment:
Stupid question: but let's assume this process gets great traction, why not `0000` instead of `9999` that we _may_ reach some day. (OK, maybe you actually want to do that because it would show a great success, "a nice problem to have" ;-))

Reply:
I like `0000` as the marker.